components/calendar/src/week_of.rs

```diff
@@ -66,7 +66,7 @@ impl WeekCalculator {
                 &provider.as_downcasting(),
                 DataRequest {
                     locale,
-                    metadata: Default::default(),
+                    ..Default::default()
```

Comment:
Thought: using `..Default::default()` means that if we ever refactor `DataRequest` again, we won't get compiler errors for missing fields, which I think was an important part of building this PR.

Reply:
Didn't really need or use that. Searching for `DataRequest {` is exhaustive because there are not constructors.